server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java

```diff
@@ -56,7 +58,9 @@
    CountdownLatchLoggingConsumer latch;
    ImageFromDockerfile image;
    private File rootDir;
+
    private final boolean preferContainerExposedPorts = Boolean.getBoolean("org.infinispan.test.server.container.preferContainerExposedPorts");
+   private final boolean deleteImageOnExit = Boolean.getBoolean("org.infinispan.test.server.container.deleteImageOnExit");
```

Comment:
This should be true by default

Reply:
In this case, do we need the argument `org.infinispan.test.server.container.deleteImageOnExit` ?

Reply:
Yeah, I don't think you're ever going to look at the image after a test execution

Reply:
Done